.github/scripts/configure-ccm.sh

```diff
@@ -25,10 +25,11 @@ function set_java_home() {
     else
         export CASSANDRA_USE_JDK11=false
     fi
-    for jdk in /opt/hostedtoolcache/Java_Adopt_jdk/${major_version}*/; 
+    for jdk in /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/${major_version}*/; 
     do 
         export JAVA_HOME="${jdk/}"x64/
         echo "JAVA_HOME is set to $JAVA_HOME"
+        export JAVA_TOOL_OPTIONS="-Dcom.sun.jndi.rmiURLParsing=legacy"
```

Comment:
question: Why was this suddenly needed? This shouldn't change the version of Java otherwise.

Reply:
I think we were running on older versions of the jdk which didn't exhibit this issue.